src/ngSanitize/sanitize.js

```diff
@@ -200,6 +200,44 @@ function $SanitizeProvider() {
     }
   };
 
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#addValidElements
+   * @kind function
+   *
+   * @param {Array|Object=} elements List of valid elements.
```

Comment:
Why the `=`? This is not optional afaict.

Reply:
Copy-paste (shame on me)